src/utils/types/ApplicationCommands.ts

```diff
@@ -1,3 +1,37 @@
+export type LocalCode =
+	| "da"
+	| "de"
+	| "en-GB"
+	| "en-US"
+	| "es-ES"
+	| "fr"
+	| "hr"
+	| "it"
+	| "lt"
+	| "hu"
+	| "nl"
+	| "no"
+	| "pl"
+	| "pt-BR "
+	| "ro"
+	| "fi"
+	| "sv-SE"
+	| "vi"
+	| "tr"
+	| "cs"
+	| "el"
+	| "bg"
+	| "ru"
+	| "uk"
+	| "hi"
+	| "th"
+	| "zh-CN"
+	| "ja"
+	| "zh-TW"
+	| "ko";
+
+export type Localization = { [key in LocalCode]?: string };
```

Comment:
Lol, testing related, forgot to remove

Reply:
Wait, I see what I did now. Okey so its because we don't currently support all of the language code in `type LocalCode` so adding the optional arguments prevents the error that says all of the lang codes should be imported.

Reply:
Why do we have the codes if we don't support them yet? Couldn't we add them as we add the language? Or have them commented out? If we can avoid having to check for undefined, would be nice imo

Reply:
By support I mean we have over 50% of the language translated on crowdin, we support all of the languages that are in the list. I just copy pasted it from the discord docs for the future of me, but we could just comment them out as they are not used

Reply:
Will comment out the parts that are not needed and make the LocalCode non optional